guides/ansible_tower/2-config/README.md

```diff
@@ -0,0 +1,172 @@
+# Exercise 2 - Configuring Ansible Tower
+
+In this exercise, we are going to configure Tower so that we can run a playbook.
+
+## The Tower UI
+
+There are a number of contructs in the Ansible Tower UI that enable multi-tenancy, notifications, scheduling, etc. However, we are only going to focus on a few of the key contructs that are required for this workshop today.
+
+* Credentials
+* Projects
+* Inventory
+* Job Template
+
+## Logging into Tower and Installing the License Key
+
+### Step 1
+
+To log in, use the username `admin` and and the password `ansibleWS`.
+
+![Ansible Tower Login Screen](ansible-lab-figure01-logon-screen.png)
+
+As soon as you login, you will prompted to request a license or browse for an existing license file
+
+![Uploading a License](at_lic_prompt.png)
+
+### Step 2
+
+In a seperate browser tab, browse to [https://www.ansible.com/workshop-license](https://www.ansible.com/workshop-license) to request a workshop license.
```

Comment:
Please add what a workshop license is -- 5 nodes for X days. (Can't recall off the top of my head.) 

Rhetorical question: should we mention and add a link to the longer-term free trial license also?